src/content/contributing/adding-wallet/index.md

```diff
@@ -0,0 +1,72 @@
+- **Open source** - your whole project’s codebase (not just modules) should be accessible and you should accept PR’s from the wider community.
+- **Non-custodial** - users control their funds. If your product disappears, users can still access and move their funds.
+- **Hardware wallet support** - users can connect their hardware wallet to sign transactions.
+- **WalletConnect** - users can connect to dapps using WalletConnect.
+- **Importing Ethereum RPC endpoints** - users can import node RPC data, allowing them to connect to a node of their choice, or other EVM compatible networks.
+- **NFTs** - users are able to view and interact with their NFTs in the wallet.
+- **Connect to Ethereum applications** - users are able to connect to and use Ethereum applications.
+- **Staking** - users are able to stake directly through the wallet.
+- **Swaps** - users are able to swap tokens through the wallet.
+- **Multichain networks** - your wallet supports users accessing multiple blockchain networks by default.
+- **Customize gas fees** - your wallet allows users to customize their transaction gas fees (base fee, priority fee, max fee).
+- **ENS support** - your wallet allows users to send transactions to ENS names.
+- **ERC-20 support** - your wallet allows users to import ERC-20 token contracts, or automatically queries and displays ERC-20 tokens.
+- **EIP-1559 (type 2) transactions** - your wallet supports EIP-1559 (type 2) transactions.
+- **Buy crypto** - your wallet supports users directly purchasing and onboarding to crypto.
+- **Withdraw crypto** - your wallet supports users withdrawing directly to card or a bank account.
+- **Multisig** - your wallet supports multiple signatures to sign a transaction.
+- **Social recovery** - your wallet supports guardians and a user can recover their wallet if they lose their seed phrase using these guardians.
+- **Dedicated support team** - your wallet has a dedicated support team where users can go to when experiencing issues.
+- **Educational resources/documentation** - your product should have a well-designed onboarding experience to help and educate users. Or evidence of how-to content like articles or videos.
+
+## Adding a wallet {#adding-a-wallet}
+
+If you want to add a wallet to ethereum.org, create an issue on GitHub.
+
+<ButtonLink to="https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=wallet+%3Apurse%3A&template=suggest_wallet.yaml&title=Suggest+a+wallet">
+  Create an issue
+</ButtonLink>
+
+## Maintenance {#maintenance}
+
+As is the fluid nature of Ethereum, teams and products come and go and innovation happens daily, so we'll undertake routine checks of our content to:
+
+- ensure that all wallets and dapps listed still fulfil our criteria
+- verify there aren't products that have been suggested that meet more of our criteria than the ones currently listed
+
```

Comment:
I'd add a note that the site is also maintained by the open source community & that we rely on the community to help keep this up to date. Then add a CTA for people to open issues/PRs if they see information is outdated!